app/controllers/dashboard_controller.rb

```diff
@@ -63,7 +63,7 @@ def iframe
       @layout = item.id if item.present?
     elsif params[:sid].present?
       item = Menu::Manager.section(params[:sid])
-      @layout = (item.items[0].id rescue nil)
+      @layout = item.items[0]&.id
```

Comment:
Not the same - `item` can be nil or `item.items` can be empty

Reply:
Well, missing `&` before `[` I guess, `items` could still be nil too...

But looking at the code..

`item` can be nil if there's no section with that id .. but assuming it exists, it should be a Menu::Section instance
`item.items` will never be nil because there's a `||= []`, it could still be a non-array but that'd just mean the section is broken
`item.items[0]` can be nil for empty sections
`item.items[0].id` will exist for any item or section

So.. yeah, `item&.items[0]&.id` seems right, it should cover the non-bug cases nicely :).